front/public/locales/de/operationalStudies/manageTrainSchedule.json

```
@@ -0,0 +1,170 @@
+{
+  "addPacedTrain": "Add a service",
+  "addVia": "Wegepunkt hinzufügen",
+  "addVias": "Wegepunkte hinzufügen",
+  "addTrainSchedule": "Zug hinzufügen",
+  "addTrainSchedules": "Einen oder mehrere Züge hinzufügen",
+  "anyTrack": "Beliebiges Gleis",
```

Same as above, is "Gleis" station-specific, or also used more out of it ? If station-specific:

```suggestion
  "anyTrack": "Beliebige Strecke",
```

---

Yep, station-specific. In this case, I'd prefer "Strecke" too.